doc/api/configuration.md

```diff
@@ -166,6 +206,28 @@ df = %sql SELECT * FROM languages
 type(df)
 ```
 
+
+## `dsn_filename`
+
+Default: `odbc.ini`
```

Comment:
Please add some references when this file is needed

Reply:
I documented this one already in a PR that has been merged